examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/provisioning/DeviceProvisioningFragment.kt

```diff
@@ -66,6 +68,8 @@ class DeviceProvisioningFragment : Fragment() {
   ): View {
     scope = viewLifecycleOwner.lifecycleScope
     deviceInfo = checkNotNull(requireArguments().getParcelable(ARG_DEVICE_INFO))
+    deviceController = ChipClient.getDeviceController(requireContext())
```

Comment:
Can you move this to `onCreate()` instead of `onCreateView()`? This way it is guaranteed to be initialized. There are some cases where a `Fragment` can be created but not create a view before going to `onDestroy()`.